### DISCUSSION TITLE
Why Cannot the Timer Be Disabled for a Scheduled Time Lapse?

### DISCUSSION THREAD

**Discussion by AlexLandherr** [gopro/labs#562] [General]
I have asked before and only gotten a very short answer, so I'll try asking again since   I just want to understand:
Why can't the timer for a scheduled time lapse or other recording be set to exactly e.g. 12:00:00?

The actual creation time of the file seems to be 12:00:03.

I was told simply that the timer "can't be disabled", would any attempt at that break other functions in the process?

**Comment by dnewman-gpsw** [comment]
In my tests the creation time was either 12:00:00 or 12:00:01, I believe the countdown is to the minute, not 3 seconds after the minute.  There is not control to disable it.  

If you use Labs commands, rather than Scheduled Capture, the start will be within a second of the target minute.    So !12:00S will shutdown start the capture as close to 12:00 as possible.

**Reply by AlexLandherr** [reply]
Ok, thanks for the clarification!

I already have a more flexible time lapse setup on a Raspberry Pi which also uses GPS Time Sync.

It uses PPS output from the GPS (in broad terms a GNSS receiver) to set the system time in Raspberry Pi OS to within a few microseconds of UTC and then my Python code uses the system time to determine whether to take a single frame of the video.

I record as still PNG images and use ffmpeg to stitch things together later on. Sadly I'm limited by the Pi HQ Camera's sensor.
